seminars/03.04.2022/try_requests.py

```diff
@@ -4,71 +4,87 @@
 import requests
 from requests import Timeout, HTTPError
 
-if __name__ == '__main__':
```

Comment:
you cannot commit changes to the seminars listings. instead move the code to the place it is expected


carefully read the docs. first part of assignment should be made within `scrapper.py` module.